packages/pwa-kit-react-sdk/src/ssr/universal/events.js

```diff
@@ -5,30 +5,26 @@
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 
-import ee from 'event-emitter'
```

Comment:
I do like that we are using native APIs as an alternative to this. but do we know why we make the decision to use `event-emitter` in the first place?

Reply:
Potentially at the time we decided to look for a solution to emit events cross-environment Node.js and browsers, EventTarget was not a thing in Node.

EventTarget is supported in Node.js since version v15.0.0.
https://developer.mozilla.org/en-US/docs/Web/API/EventTarget#browser_compatibility